### DISCUSSION TITLE
How to clean argo rollout objects ?

### DISCUSSION THREAD

**Discussion by JeanYvesQueignec** [argoproj/argo-rollouts#1271] [Q&A] [answered]
Hello,
I start to look at Argo Rollout (as I have already investigated on ArgoCD )
I made some trials and test different Rollouts but some are not correct and I get a  list using  command;
kubectl argo rollouts list rollouts

How to clean the list to remove the one I do not want anymore ?

Regards,
Thanks,
Jean-Yves

**Comment by jessesuen** [comment, marked as answer]
You can just use delete functionality of kubectl (no need for plugin):

```
kubectl delete ro ROLLOUTNAME
```

**Reply by JeanYvesQueignec** [reply]
OK clear thanks !
Regards,
Jean-Yes
